src/backends/plonky2/circuits/mainpod.rs

```diff
@@ -1612,41 +1617,6 @@ impl InnerCircuit for MainPodVerifyTarget {
         for (i, (st, op)) in zip_eq(&input.statements, &input.operations).enumerate() {
             self.input_statements[i].set_targets(pw, &self.params, st)?;
             self.operations[i].set_targets(pw, &self.params, op)?;
-            if matches!(
```

Comment:
This logic is now handled in `src/backends/plonky2/mainpod/mod.rs` `extract_public_key_of`